src/h_vialib/client.py

```diff
@@ -0,0 +1,62 @@
+"""Helper classes for clients using Via proxying."""
+
+import re
+from urllib.parse import urlencode, urlparse
+
+
+class ViaDoc:  # pylint: disable=too-few-public-methods
+    """A doc we want to proxy with content type."""
+
+    GOOGLE_DRIVE_REGEX = re.compile(
+        r"^https://drive.google.com/uc\?id=(.*)&export=download$", re.IGNORECASE
+    )
```

Comment:
Recent feedback on one of my reviews was to make this internal with `_` at the front